src/DucksNet.API/Controllers/EmployeesController.cs

```diff
@@ -60,37 +60,42 @@ public IActionResult Create([FromBody] EmployeeDTO dto)
                 return BadRequest("The telephone number already exists");
```

Comment:
Doesn't the Create(...) method already return an error?

Reply:
here checks if the email/telephone number already exists in the database. I need to querry the database and has to be here.